app/templates/email/en_us/misc/password_reset.jinja2

```diff
@@ -0,0 +1,9 @@
+<html>
+    <body>
+        <p>Hello {{ site_name }} user,<p>
+
+        <p>You can reset your password here: <a href="{{ reset_link }}">{{ reset_link }}</a></p>
+
```

Comment:
wonder if we need something to the effect of:

> if you received this without requesting a password reset, you can ignore it.

or something like that -- which a lot of password reset emails will include.

Reply:
Yeah I thought about that too, I'll add it